src/supertux/screen_manager.cpp

```diff
@@ -181,6 +181,18 @@ struct ScreenManager::FPS_Stats
   float get_fps_min() const { return last_fps_min; }
   float get_fps_max() const { return last_fps_max; }
 
+  // This returns the highest measured delay between two frames from the
+  // previous and current 0.5 s measuring intervals
+  float get_highest_max_ms()
```

Comment:
Can you mark this const? It doesn't appear to have side effects.

Reply:
done